mteb/evaluation/MTEB.py

```diff
@@ -10,7 +10,7 @@
 
 from .. import __version__
 from ..abstasks import *
-from ..abstasks import AbsTask
+from ..abstasks import AbsTask, LangMapping
```

Comment:
I don't think we should modify the functionning of the MTEB package, can we do this preprocessing in a script, because it's very specific to this dataset ? See `mteb/scripts/data/`. Another possibility is to upload the preprocessed dataset to HF (with reference to the original authors)

Reply:
MTEB have used bigrams fr, en, pl, which often correspond to countries, except that there isn't one language per country, and many languages have no associated countries.
In the Masakhane dataset, there are 16 languages in trigrams, the majority of which do not correspond to a country. And this won't be the only dataset. There's already another one in MTEB, the Tatoeba for the BitextMining task, which isn't even used in their English script. They had to run it manually maybe.
As a result, the little code I've added is used when the script is run on "fr", adding "fra" for example. Their code already includes an extension from "fr" to "fr-fr".
Moreover i don't think uploading dataset in different format is very good.